gateway-service/src/main/java/com/example/gatewayservice/common/exception/ErrorCode.java

```diff
@@ -10,7 +10,8 @@ public enum ErrorCode {
     NEED_RE_LOGIN(401, 2501, "로그인을 다시 해주세요."),
     NEED_RE_ISSUE(401, 2502, "AccessToken을 재발행해주세요."),
     NEED_SIGNUP(403, 2503, "회원가입이 필요합니다."),
-    FORBIDDEN(403, 2504, "요청을 수행 할 권한이 부족합니다.");
+    FORBIDDEN_CLIENT(403, 2504, "CLIENT 권한이 필요한 요청입니다. 권한 등록 이후 시도해주세요."),
+    FORBIDDEN_FREELANCER(403, 2505, "FREELANCER 권한이 필요한 요청입니다. 권한 등록 이후 시도해주세요.");
```

Comment:
FORBIDDEN_`ROLE`이라는 표현을 해석하는 과정에서 오해가 발생할 수 있을 것 같아요👓

- 의도: 요청에 `ROLE` 권한이 필요하지만, 해당 권한이 없는 사용자가 요청했을 때 발생하는 예외

- 가능: `ROLE` 권한을 가진 사용자가 요청했을 때 발생하는 예외

<br>

의도를 더 명확하게 표현하기 위해 아래와 같은 코드를 사용하는 것은 어떨까요?🤓

```suggestion
    REQUIRED_CLIENT_ROLE(403, 2504, "CLIENT 권한이 필요한 요청입니다. 권한 등록 이후 시도해주세요."),
    REQUIRED_FREELANCER_ROLE(403, 2505, "FREELANCER 권한이 필요한 요청입니다. 권한 등록 이후 시도해주세요.");
```

Reply:
명확한 의도의 네이밍이 항상 고민이네요..
지수님이 매번 알려주셔서 어떻게 하면 더 이해하기 좋은 네이밍이 될 수 있는지 배워갑니다.
감사합니다!

수정해두도록 하겠습니다!

Reply:
최신 Refactor PR 에 반영하였습니다.